Mathlib/CategoryTheory/Sites/Coherent/RegularSheaves.lean

```diff
@@ -36,7 +36,7 @@ class Presieve.regular {X : C} (R : Presieve X) : Prop where
   single_epi : ∃ (Y : C) (f : Y ⟶ X), R = Presieve.ofArrows (fun (_ : Unit) ↦ Y)
     (fun (_ : Unit) ↦ f) ∧ EffectiveEpi f
 
-namespace regularCoverage
+namespace regularTopology
```

Comment:
I prefer regularCoverage: regularTopology seems too much related to "true" topology.

Reply:
`regularTopology` is consistent with `coherentTopology` and `extensiveTopology`. Do you not like those either or is it just this one because of the notion of regular topological space?

Reply:
An obvious advantage of using `regularTopology` is that `regularTopology C` is a lot shorter than `(regularCoverage C).toGrothendieck`. 

Also, it's in the `CategoryTheory` namespace, which avoids confusion with general topology.

Reply:
I agree `regularTopology` is better: writing
```lean
IsSheaf (regularTopology C) F
```
is nice.